back/board/src/main/java/com/codingrecipe/board/controller/BoardApiController.java

```diff
@@ -0,0 +1,22 @@
+package com.codingrecipe.board.controller;
+
+import com.codingrecipe.board.dto.BoardDTO;
+import com.codingrecipe.board.service.BoardService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("api/board")
+public class BoardApiController {
+    private final BoardService boardService;
+
+    @GetMapping
+    public List<BoardDTO> get() {
```

Comment:
```suggestion
    public List<BoardDTO> get() {
```
`BoardDTO`는 현재 save에서도 사용하는것으로 보여지며 frontend에서 사용되지 않는값도 포함되어있는것으로 보여집니다.
API Response로 사용하는 DTO를 따로 만드시는것을 추천드립니다.